src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '0.91.0'
+__version__ = '0.91.0'
```

Comment:
What's up with the no-trailing-newline?

Reply:
Just a typo when resolving conflicts. Fixed